internal/engine/source/activitypub/option.go

```diff
@@ -18,7 +17,7 @@ type Option struct {
 func NewOption(n network.Network, parameters *config.Parameters) (*Option, error) {
 	var option Option
 
-	fmt.Println("parameters is:", parameters)
+	zap.L().Info("parameters is:", zap.Any("parameters", parameters))
```

Comment:
There are some repetitions in the log information, which will form a strange sentence like `parameters is parameters: `. And it should use `are` I think.